mlflow/pipelines/regression/v1/pipeline.py

```diff
@@ -159,7 +171,19 @@ class RegressionPipeline(_BasePipeline):
         regression_pipeline.inspect(step="evaluate")
     """
 
-    _PIPELINE_STEPS = (IngestStep, SplitStep, TransformStep, TrainStep, EvaluateStep, RegisterStep)
+    _PIPELINE_STEPS = (
+        # Batch scoring DAG
+        IngestScoringStep,
+        PredictStep,
+        # Training data ingestion DAG
+        IngestStep,
+        # Model training DAG
+        SplitStep,
+        TransformStep,
+        TrainStep,
+        EvaluateStep,
+        RegisterStep,
+    )
```

Comment:
With this setup, does this mean that when the entire MLP regression pipeline is run, all of the steps, including the Batch Scoring DAG will be executed?

Reply:
No.  I asked about this in a thread in channel.  For now, we can leave this as is and Batch Scoring will need to be explicitly called.